packages/core/src/odata/v2/filter-functions.ts

```diff
@@ -36,11 +42,29 @@ export function replace<EntityT extends Entity>(
   return filterFunction('replace', 'string', str, searchStr, replaceStr);
 }
 
+/* Export some filter functions for backwards compatibility */
+export function length<EntityT extends EntityBase>(
+  str: string | Field<EntityT> | StringFilterFunction<EntityT>
+): NumberFilterFunction<EntityT> {
+  return filterFunctionsCommon.length(str);
+}
+
+/* Export some filter functions for backwards compatibility */
+export function substring<EntityT extends EntityBase>(
```

Comment:
I think you could make your life easier by doing this:
```
/**
 * @hidden
 * Export some filter functions for backwards compatibility
 */
export const substring = filterFunctionsCommon.substring;
```

Also hide from the documentation ;)

Reply:
👍